moduleframework/dockerlinter.py

```diff
@@ -286,6 +286,9 @@ def check_copy_files_exist(self):
         dir_name = os.getcwd()
         files = self._get_copy_add_files(os.path.dirname(self.dockerfile))
         f_exists = False
+        if not files:
+            f_exists = True
+            return f_exists
```

Comment:
I suggest:
to use there ``return True`` or maybe better will be to have there something like:
```
if not files:
    f_exists = True
else:
    for f in files:
        if f.startswith('http'):
    ...
return  f_exists
```